packages/dashboard/src/app/views/editorSettings/archive/index.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useCallback } from '@web-stories-wp/react';
+import { __, _x, TranslateWithMarkup, sprintf } from '@web-stories-wp/i18n';
+import { useFeature } from 'flagged';
+import { DropDown, THEME_CONSTANTS } from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+
+import {
+  InlineLink,
+  SettingForm,
+  SettingHeading,
+  TextInputHelperText,
+} from '../components';
+import { ARCHIVE_TYPE } from '../../../../constants';
+
+export const TEXT = {
+  LABEL: __('Stories Archives', 'web-stories'),
+  SECTION_HEADING: __('Stories Archives', 'web-stories'),
+  ARCHIVE_CONTENT: __(
+    "By default WordPress automatically creates an archive page, displaying your latest stories in your theme's layout.",
+    'web-stories'
+  ),
+  SUB_TEXT_DISABLED: __('Disabled the default archive page.', 'web-stories'),
+};
+
+const OPTIONS = [
+  {
+    label: _x('Default', 'archive type', 'web-stories'),
+    value: ARCHIVE_TYPE.DEFAULT,
+  },
+  {
+    label: __('Disabled', 'web-stories'),
+    value: ARCHIVE_TYPE.DISABLED,
+  },
+];
+
+export default function ArchiveSettings({
+  archive = 'default',
+  archiveURL,
+  updateSettings,
+}) {
+  const onChange = useCallback(
+    (_, newArchive) => updateSettings({ archive: newArchive }),
+    [updateSettings]
+  );
+
+  const isFeatureEnabled = useFeature('disableArchive');
+
+  if (!isFeatureEnabled) {
+    return null;
+  }
+
+  return (
+    <SettingForm>
+      <div>
+        <SettingHeading>{TEXT.SECTION_HEADING}</SettingHeading>
+        <TextInputHelperText
+          size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+        >
+          {TEXT.ARCHIVE_CONTENT}
+        </TextInputHelperText>
+      </div>
+      <div>
+        <DropDown
+          ariaLabel={TEXT.LABEL}
+          options={OPTIONS}
+          selectedValue={archive}
+          onMenuItemClick={onChange}
+          fillWidth
+        />
```

Comment:
I'll make a new ticket to follow this up, but the DropDown isn't working for me here unless you wrap it in a div tag of it's own. I think it's something to do with overflow. Weird that this didn't come up at all anywhere else. 

```suggestion
<div>
        <DropDown
          ariaLabel={TEXT.LABEL}
          options={OPTIONS}
          selectedValue={archive}
          onMenuItemClick={onChange}
          fillWidth
        />
        </div>
```

Reply:
Can you show me what is the issue is here?

Reply:
weird, i swear this was an issue last week. It's totally fine now though which is good because I was very confused why this was a thing.